pymc/tests/test_distributions.py

```diff
@@ -1154,6 +1154,9 @@ def test_wald_logp_custom_points(self, value, mu, lam, phi, alpha, logp):
         decimals = select_by_precision(float64=6, float32=1)
         assert_almost_equal(model.fastlogp(pt), logp, decimal=decimals, err_msg=str(pt))
 
+    @pytest.mark.xfail(
+        reason="Fails because mu and sigma values are being picked randomly from domains"
+    )
```

Comment:
From the text here I don't understand what this means.

Can you open a corresponding issue?

Reply:
There's already an issue running regarding this: https://github.com/pymc-devs/pymc3/issues/5023